xml/resources-3.10.rng

```diff
@@ -331,11 +346,25 @@
               <define name="element-nvset.name">
                 <parentRef name="op.meta_attributes.nvpair.name-unsupported"/>
               </define>
+
+              <!--
+               @COMPAT: Support for node attribute expressions is deprecated
+               here. We can just delete this define when we drop support.
+              -->
+              <define name="element-nvset.rule">
+                <parentRef name="element-rule-node-allowed"/>
+              </define>
             </include>
           </grammar>
         </element>
         <element name="instance_attributes">
```

Comment:
Yeah, that's a minefield, but unfortunately it's needed for `trace_ra` (which should really be a meta-attribute). It would be absurd to use it for anything else.

Reply:
and OCF_CHECK_LEVEL